build/yamls/antrea-aks.yml

```diff
@@ -1000,12 +1001,20 @@ data:
     # Default MTU to use for the host gateway interface and the network interface of each Pod.
     # If omitted, antrea-agent will discover the MTU of the Node's primary interface and
     # also adjust MTU to accommodate for tunnel encapsulation overhead (if applicable).
+    # 1450 is the default value for IPv4. With IPv6 setup, its value is usually 50 less than IPv4 one
```

Comment:
I think IPv6 header is just 20 bytes more than v4?

And this only applies to IPv6 tunnel right? Then consider: 1450 is the default value for IPv4 Geneve or VXLAN encap mode; the default value for IPv6 Geneve or VXLAN encap mode is 1430.

Reply:
Make sense. Updated 50 -> 20.